spec/rubocop/cop/chef/style/negating_only_if_spec.rb

```diff
@@ -35,19 +35,36 @@
     RUBY
   end
 
-  it 'does not register an offense when an only_if does not negate ruby' do
+  it 'does not register an offense when an only_if double negates a variable' do
     expect_no_offenses(<<~RUBY)
+      foo = true
       package 'legacy-sysv-deps' do
-        only_if { foo }
+        only_if { !!foo }
       end
     RUBY
   end
 
-  it 'does not register an offense when an only_if negates within the ruby' do
+  it 'does not register an offense when an only_if double negates' do
```

Comment:
Took me a while to see why this wasn't a dup. Maybe make the title a "double negates a non-variable" ?

Reply:
yeah it's not super clear. Spelling out the full description of what some of these does ends up being a paragraph. Let me clean that up